app/src/androidTest/java/org/mozilla/fenix/ui/SettingsPrivacyTest.kt

```diff
@@ -616,15 +619,73 @@ class SettingsPrivacyTest {
             clickDeleteBrowsingDataButton()
             confirmDeletionAndAssertSnackbar()
             verifyBrowsingHistoryDetails("0")
-        }.goBack {
-            verifyGeneralHeading()
-        }.goBack {
+            exitMenu()
         }
         navigationToolbar {
         }.openThreeDotMenu {
         }.openHistory {
             verifyEmptyHistoryView()
+            mDevice.pressBack()
+        }
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(storageCheckPage) {
+            verifyPageContent("Session storage empty")
+            verifyPageContent("Local storage empty")
+        }
+    }
+
+    @SmokeTest
+    @Test
+    fun deleteCookiesTest() {
+        val cookiesTestPage = getStorageTestAsset(mockWebServer, "storage_write.html").url
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(cookiesTestPage) {
+            verifyPageContent("No cookies set")
+            clickSetCookiesButton()
+            verifyPageContent("user=android")
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openSettingsSubMenuDeleteBrowsingData {
+            selectOnlyCookiesCheckBox()
+            clickDeleteBrowsingDataButton()
+            confirmDeletionAndAssertSnackbar()
+            exitMenu()
+        }
+        browserScreen {
+        }.openThreeDotMenu {
+        }.refreshPage {
+            verifyPageContent("No cookies set")
+        }
+    }
+
+    @SmokeTest
+    @Test
+    fun deleteCachedFilesTest() {
+        homeScreen {
+            verifyExistingTopSitesTabs("Top Articles")
+        }.openTopSiteTabWithTitle("Top Articles") {
+            waitForPageToLoad()
+        }.openTabDrawer {
+        }.openNewTab {
+        }.submitQuery("about:cache") {
+            // disabling wifi to prevent downloads in the background
+            setNetworkEnabled(enabled = false)
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openSettingsSubMenuDeleteBrowsingData {
+            selectOnlyCachedFilesCheckBox()
+            clickDeleteBrowsingDataButton()
+            confirmDeletionAndAssertSnackbar()
+            exitMenu()
+        }
+        browserScreen {
+        }.openThreeDotMenu {
+        }.refreshPage {
+            verifyNetworkCacheIsEmpty("memory")
+            verifyNetworkCacheIsEmpty("disk")
         }
+        setNetworkEnabled(enabled = true)
     }
```

Comment:
Just double checking if most of these strings in these 3 tests can be queried by resource?

Reply:
done